packages/fxa-shared/guards/AdminPanelGuard.ts

```diff
@@ -15,11 +21,14 @@ export enum PermissionLevel {
 export enum AdminPanelFeature {
   AccountLogs = 'AccountLogs',
   AccountSearch = 'AccountSearch',
-  ConnectedServices = 'ActiveSessions',
+  AccountHistory = 'AccountHistory',
+  ConnectedServices = 'ConnectedServices',
```

Comment:
🎉  This feels a lot clearer than the previous `ConnectedServices = 'ActiveSessions',`